src/engine/sysDef.cpp

```diff
@@ -1883,6 +1883,15 @@ void DivEngine::registerSystems() {
     {DIV_INS_AMIGA}
   );
 
+  sysDefs[DIV_SYSTEM_YMZ280B]=new DivSysDef(
+    "Yamaha YMZ280B", NULL, 0xb8, 0, 8, false, true, 0x151, false,
+    "used in some arcade boards. Has 8 channels of either 4-bit ADPCM, 8-bit PCM or 16-bit PCM.",
```

Comment:
Don't specify channels - this will be displayed to the user automatically in the upcoming system selector.